bluemira/codes/process/_inputs.py

```diff
@@ -0,0 +1,303 @@
+# bluemira is an integrated inter-disciplinary design tool for future fusion
+# reactors. It incorporates several modules, some of which rely on other
+# codes, to carry out a range of typical conceptual fusion reactor design
+# activities.
+#
+# Copyright (C) 2021 M. Coleman, J. Cook, F. Franza, I.A. Maione, S. McIntosh, J. Morris,
+#                    D. Short
+#
+# bluemira is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# bluemira is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with bluemira; if not, see <https://www.gnu.org/licenses/>.
+"""
+Parameter classes/structures for Process
+"""
+
+from dataclasses import dataclass, field, fields
+from typing import Dict, Generator, List
+
+from bluemira.codes.process.api import INVariable
+
+
+@dataclass
+class ProcessInputs:
+    """
+    Process Inputs dataclass
+
+    Notes
+    -----
+    All entries get wrapped in an INVariable class to enable easy InDat writing
+    """
+
+    bounds: Dict[str, Dict[str, str]] = field(
+        default_factory=lambda: {
+            "2": {"u": "20.0"},
+            "3": {"u": "13"},
+            "4": {"u": "150.0"},
+            "9": {"u": "1.2"},
+            "18": {"l": "3.5"},
+            "29": {"l": "0.1"},
+            "38": {"u": "1.0"},
+            "39": {"u": "1.0"},
+            "42": {"l": "0.05", "u": "0.1"},
+            "50": {"u": "1.0"},
+            "52": {"u": "10.0"},
+            "61": {"l": "0.02"},
+            "103": {"u": "10.0"},
+            "60": {"l": "6.0e4", "u": "9.0e4"},
+            "59": {"l": "0.50", "u": "0.94"},
+        }
+    )
+    # fmt: off
+    icc: List[int] = field(default_factory=lambda: [1, 2, 5, 8, 11, 13, 15, 16, 24, 25,
+                                                    26, 27, 30, 31, 32, 33, 34, 35, 36,
+                                                    60, 62, 65, 68, 72])
+    ixc: List[int] = field(default_factory=lambda: [2, 3, 4, 5, 6, 9, 13, 14, 16, 18,
+                                                    29, 36, 37, 38, 39, 41, 42, 44, 48,
+                                                    49, 50, 51, 52, 53, 54, 56, 57, 58,
+                                                    59, 60, 61, 102, 103, 106, 109, 110,
+                                                    113, 117, 122, 123])
+    # fmt: on
+    abktflnc: float = 15.0
```

Comment:
Ahh PROCESS, this is a real research science variable name 😆. Can't even begin to think what this means.

Reply:
Bit of a pain, I know, but should these have units 😨? Or we, at least, need a something in the docstring(s) that says where to find the units

Reply:
For all of these the units are in the mappings or they live in a process dictionary. The process units are a whole other issue...they've been working on it for a while

Reply:
added a bit to the description